packages/containers/psammead-timestamp-container/src/index.jsx

```diff
@@ -18,17 +18,21 @@ const TimestampContainer = ({
   suffix,
   timezone,
   script,
+  locale,
 }) => {
   if (!isValidDateTime(new Date(timestamp))) {
     return null;
   }
 
+  const datetime = formatUnixTimestamp(
+    timestamp,
+    dateTimeFormat,
+    timezone,
+    locale,
+  );
+
   return (
-    <Timestamp
-      datetime={formatUnixTimestamp(timestamp, dateTimeFormat, timezone)}
-      padding={padding}
-      script={script}
-    >
+    <Timestamp datetime={datetime} padding={padding} script={script}>
       {prefix ? `${prefix} ` : null}
       {showRelativeTime(timestamp, isRelative, format, timezone)}
```

Comment:
Do you not need to pass locale to showRelativeTime?

Reply:
TBH I prefer setting it once at the top of the container, avoids issues of having to pass it around everywhere